mapeditor/mainwindow.ui

```diff
@@ -734,7 +734,7 @@
           <x>0</x>
           <y>0</y>
           <width>128</width>
-          <height>251</height>
+          <height>257</height>
```

Comment:
why size change is needed?

Reply:
that's a standard thing when you open UI file in another OS - it's rendered with native look which can be different in size (e.g. macOS vs KDE)

Reply:
This ^
The only intended changes in .ui files were:
- marking strings as non-localizeable
- removing some "internal" strings, e.g. FormName, ComboBox fron titles